pushy/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>pushy</artifactId>
+    <name>Pushy</name>
+
+    <parent>
+        <groupId>com.relayrides</groupId>
+        <artifactId>pushy-parent</artifactId>
+        <version>0.7.2-SNAPSHOT</version>
+    </parent>
+
+    <dependencies>
```

Comment:
It might be that some are being used on the other moduels, in that case I'd move them up, again .